src/shared/ui/short-info-panel/short-info-panel.tsx

```
@@ -3,36 +3,50 @@
 
 import { DateSimple } from '../date-simple';
 import micIcon from '@/shared/assets/icons/mic.svg';
-import { LABELS } from '@/shared/constants';
-import { calculateFreshDate } from '@/shared/helpers/get-course-date';
+import { LABELS, TO_BE_DETERMINED } from '@/shared/constants';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)